automatic/flashplayeractivex/update.ps1

```diff
@@ -1,7 +1,9 @@
-
+﻿
 import-module au
+. "$PSScriptRoot\..\..\scripts\Get-Padded-Version.ps1"
 
 $releases = "https://get.adobe.com/en/flashplayer/" # URL to for GetLatest
+$padVersionUnder = '24.0.1'
```

Comment:
@AdmiringWorm nope, I think this makes sense.  I just wanted to make sure that I wasn't missing something.  The docs you have added really help with understanding.